client/asset/dcr/dcr.go

```diff
@@ -449,7 +441,7 @@ type findRedemptionResult struct {
 
 // NewWallet is the exported constructor by which the DEX will import the
 // exchange wallet.
-func NewWallet(cfg *asset.WalletConfig, logger dex.Logger, network dex.Network) (asset.Wallet, error) {
+func NewWallet(cfg *asset.WalletConfig, logger dex.Logger, network dex.Network) (*ExchangeWallet, error) {
```

Comment:
I just reverted the `baseWallet` commit and this was part of it.  Only part of that commit I didn't revert was the syntax change to `feeSuggestionAny`.

Anyway, I sorta prefer having `NewWallet` return a concrete type tbh.  Just having `(*Driver).Open` wrapping in the interface is fine IMO.  Either was is fine to me.

I do believe this package should have an exported constructor like `NewWallet` even though we're using the driver system primarily, mainly for completeness, otherwise it can be an awkward package to use.  I think I have used `NewWallet` in some random apps I have hacked on when I am just importing the dcr package and don't need to mess with the Driver system.

Reply:
Ok, sounds good to me. Just was wondering, I see no problem here.